raijin_scripts/test_deaenv_stats/dea_testscripts/cp_statsyaml_files.sh

```diff
@@ -25,7 +26,24 @@ do
   yaml_filename=$(basename "$FC_STATS_CONF_DIR")
 
   wget -q "$FC_STATS_CONF_DIR"
-  sed -e 's,location: .*,location: "'"$WORKDIR"'/output_files/stats/fc/001",' "$yaml_filename" > "$yaml_filename.tmp" && mv "$yaml_filename.tmp" "$yaml_filename"
+  sed -e 's,location: .*,location: "'"$WORKDIR"'/fc_stats/001",' "$yaml_filename" > "$yaml_filename.tmp" && mv "$yaml_filename.tmp" "$yaml_filename"
```

Comment:
Just use `sed -i -e` here to edit in place, it'll make the code much shorter and clearer, and the whole script will fail anyway if something goes wrong.